### DISCUSSION TITLE
Transfer-Encoding : chunked and content-length both set

### DISCUSSION THREAD

**Discussion by plpioko** [quarkusio/quarkus#40778] [Q&A]
Hi,
I am quite new to Quarkus and use Quarkus version 3.10.0. I am trying to develop Rest Client app that is capable of generating and sending files based on received data. Therefore I decided to use QuarkusRestClientBuilder class as well ClientMultipartForm:
`APIintrface apiinterface = QuarkusRestClientBuilder.newBuilder().baseUri(URI.create(BASE_URL)).register(new ReqFilter(token)).build(APIintrface.class);`
....
`            ClientMultipartForm multiPartForm = ClientMultipartForm.create();
            multiPartForm.attribute("json_data", jsonData, "json_data");
            multiPartForm.binaryFileUpload("pdffile", "pdffile.pdf", "src/main/resources/code64.cdf" , MediaType.APPLICATION_OCTET_STREAM);
            k = apiinterface.multipartInterface(multiPartForm);`
where my interface is:
`    @POST
    @Consumes(MediaType.MULTIPART_FORM_DATA)
    @Produces(MediaType.APPLICATION_JSON)
    //@PartType(MediaType.TEXT_PLAIN) 
    String multipartInterface(ClientMultipartForm dataParts);`

The problem is that if I attach file then my request generates both 
> transfer-encoding  chunked
> content-length 11111
headers which my other servers do not accept (thankfully I could spot it using webhook.site). Is there any way to prevent that both of those are created or there is something wrong with my code?

Reository: [https://github.com/plpioko/rest_playground](https://github.com/plpioko/rest_playground)

**Comment by gsmet** [comment]
@plpioko given you have a reproducer, I think it's worth creating an issue. Is your server something exotic or not? In any case, that's really something we need to evaluate as ideally we should work with any server.
